src/cbc_sdk/rest_api.py

```diff
@@ -251,8 +283,7 @@ def _bulk_threat_update_status(self, threat_ids, status, remediation, comment):
 
     def bulk_threat_update(self, threat_ids, remediation=None, comment=None):
         """
-        Update the alert status of alerts associated with multiple threat IDs.
-        The alerts will be left in an OPEN state after this request.
+        Update the alert status of alerts associated with multiple threat IDs. The alerts will be left in an OPEN state
 
         :param threat_ids list: List of string threat IDs.
```

Comment:
These are not in google docstring format, same comment for `bulk_threat_dismiss()` and `fetch_process_queries()`

Reply:
Flake8 just cares if the docstring is present and follows certain guidelines (summary line has a blank line following, etc.) If I had tried to reformat all the docstrings into Google format this would have taken _much_ longer. Perhaps we need another ticket to address reformatting all non-Google docstrings?

Reply:
Ahh I didn't know that. We should probably have a story for converting docstrings to Google format @kebringer-cb